libraries/aws_s3_bucket_policy.rb

```diff
@@ -33,6 +33,6 @@ def resource_id
   end
 
   def to_s
-    "S3 Bucket Name: #{@display_name}"
+    "S3 Bucket Policy #{@display_name}"
```

Comment:
We can't change 'Name' to 'Policy' as @display_name returns the name of the bucket.

Reply:
but  for now the description  of POLICY returns BUCKET name

Reply:
Then we have to change the value of the @display_name.